pay/transactie.php

```diff
@@ -6,9 +6,6 @@
     <head>
         <meta charset="utf-8">
         <title>Betalen</title>
-        <?php
-        include __DIR__ . "../header.php";
```

Comment:
Hier include je de header in de header. Dubbel op dus. De header.php heeft zijn eigenhtml waar de <head> in staat.

De bedoeling is dat je de header include, je content maakt en dan de footer include

- Header
- Eigen content
- Footer

Reply:
Worked, done